spec/requests/workers_spec.rb

```diff
@@ -16,49 +40,40 @@ def tickets_count(worker)
       amount
     end
 
-    it 'return a success response index' do
-      get '/workers'
-      expect(response).to have_http_status(200)
-    end
-
-    it 'return a success create' do
-      post "/workers", :params => { :data => {"first_name" => worker.first_name,
-                                              "last_name"=> worker.last_name,
-                                              "age"=> worker.age,
-                                              "role"=> worker.role} }, :headers => HEADERS
-
-      expect(response.content_type).to eq("application/json; charset=utf-8")
-      expect(response).to have_http_status(:created)
-    end
-
-    context "when ticket exist" do
-      before do
-        worker
-      end
-
+    context "when worker exist" do
       it 'return valid json index' do
         get'/workers'
         body = JSON.parse(response.body)
```

Comment:
I think you already impemented helper like `json_response` or smth like it?

Reply:
blueprint?